Sources/FluentUI_macOS/Components/MultilinePillPicker/MultilinePillPicker.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// This is a work-in-progress control for hosting multiple rows of pill buttons. At present, this control
+/// only supports a hard-coded two rows of elements.
```

Comment:
Do we need to say "work in progress" here?

Reply:
My hope is that this would be a helpful heads-up for downstream consumers of the library, since we don't currently have a formal way of marking a component as "experimental". But I can remove it if you think it might be more confusing than helpful.

Reply:
Eh, fair point. Up to you

Reply:
Done!